DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces_overview/host-pipes/Readme.md

```diff
@@ -0,0 +1,253 @@
+# Host Pipe Streaming
+This implementation uses a register-mapped invocation interface, and demonstrates how to use host pipes to configure a streaming data interface.
```

Comment:
Here I think we should coordinate between this sample and the host pipes sample to try to avoid duplication (either have that sample refer to this one or refer to that sample here)

Reply:
I agree with William, and I'd say that we may want to remove the host pipes sample entirely if this one covers the same topic